runtimes/polimec/src/lib.rs

```diff
@@ -1085,7 +1085,7 @@ impl pallet_linear_release::Config for Runtime {
 	type UnvestedFundsAllowedWithdrawReasons = shared_configuration::vesting::UnvestedFundsAllowedWithdrawReasons;
 	type WeightInfo = pallet_linear_release::weights::SubstrateWeight<Runtime>;
 
-	const MAX_VESTING_SCHEDULES: u32 = 26;
+	const MAX_VESTING_SCHEDULES: u32 = 100;
```

Comment:
Can we increase this to 100? Were we not running into some max value?

Reply:
The problem is when the max schedules is lower than the number of total contributions+bids, since we won't be able to settle participations till we merge schedules. Until we implement an automatic schedule merge, we'll increase this limit